pgjdbc/src/test/java/org/postgresql/test/jdbc2/Jdbc2TestSuite.java

```diff
@@ -35,99 +35,82 @@
  */
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
-        ANTTest.class,
-        JavaVersionTest.class,
-
-        DriverTest.class,
-        ConnectionTest.class,
-        DateStyleTest.class,
-        DatabaseMetaDataTest.class,
-        DatabaseMetaDataPropertiesTest.class,
-        SearchPathLookupTest.class,
-        EncodingTest.class,
-        ExpressionPropertiesTest.class,
-        ColumnSanitiserDisabledTest.class,
-        ColumnSanitiserEnabledTest.class,
-        LruCacheTest.class,
-        HostSpecTest.class,
-        ReaderInputStreamTest.class,
-        ServerVersionParseTest.class,
-        ServerVersionTest.class,
-
-        OptionsPropertyTest.class,
-
-        TypeCacheDLLStressTest.class,
-
-        ResultSetTest.class,
-        ResultSetMetaDataTest.class,
-        StringTypeUnspecifiedArrayTest.class,
-        ArrayTest.class,
-        PrimitiveArraySupportTest.class,
-        RefCursorTest.class,
-
-        DateTest.class,
-        TimeTest.class,
-        TimestampTest.class,
-        TimezoneTest.class,
-        PGTimeTest.class,
-        PGTimestampTest.class,
-        TimezoneCachingTest.class,
-        ParserTest.class,
-        ReturningParserTest.class,
-        CommandCompleteParserTest.class,
-        CommandCompleteParserNegativeTest.class,
-        ReplaceProcessingTest.class,
-
-        OidToStringTest.class,
-        OidValueOfTest.class,
-
-        PreparedStatementTest.class,
-        StatementTest.class,
-        QuotationTest.class,
-
-        ServerPreparedStmtTest.class,
-
-        BatchExecuteTest.class,
-        BatchFailureTest.class,
-
-        BatchedInsertReWriteEnabledTest.class,
-        NativeQueryBindLengthTest.class,
-        DeepBatchedInsertStatementTest.class,
-        JBuilderTest.class,
-        MiscTest.class,
-        NotifyTest.class,
-        DatabaseEncodingTest.class,
-        ClientEncodingTest.class,
-
-        BlobTest.class,
-        BlobTransactionTest.class,
-
-        UpdateableResultTest.class,
-
-        CallableStmtTest.class,
-        CursorFetchTest.class,
-        ConcurrentStatementFetch.class,
-        ServerCursorTest.class,
-
-        IntervalTest.class,
-        GeometricTest.class,
-
-        LoginTimeoutTest.class,
-        TestACL.class,
-
-        ConnectTimeoutTest.class,
-
-        PGPropertyTest.class,
-
-        V3ParameterListTests.class,
-
-        CopyTest.class,
-        CopyLargeFileTest.class,
-        ServerErrorTest.class,
-        UpsertTest.class,
-        OuterJoinSyntaxTest.class,
-
-        UTF8EncodingTest.class
```

Comment:
This test needed to be kept in the list...

Reply:
hmmm ok, we can add it to this PR

Reply:
I have added it back.

Reply:
Ok, lets let travis complete and I'll push it